src/api/authentication.js

```diff
@@ -26,25 +27,51 @@ const auditingExemptPaths = [
 
 const isUndefOrEmpty = string => (string == null) || (string === '')
 
-export async function authenticate (ctx, next) {
+async function authenticateBasic (ctx) {
+  const credentials = basicAuth(ctx)
+  if (credentials == null) {
```

Comment:
Is the use of `==` intentional? Automatic type casting doesn't read well and is generally not good style. Even though I don't see a problem with it for null checks.

Reply:
Yes it's intentional. I want to check for `null` or `undefined` but not other falsy values in this case.

Reply:
It would be better to be explicit them for readability, but not a big issue.